safe/gis/vector/prepare_vector_layer.py

```diff
@@ -234,7 +237,8 @@ def rename_remove_inasafe_fields(layer):
     # Remove useless fields
```

Comment:
Just a note for future, I noticed  the way you generally use the word 'useless' - it is better phrased 'unnecessary'. Useless is often derogatory 'good for nothing' versus unnecessary meaning 'not needed'....